### DISCUSSION TITLE
Kill newsfeed. Its not useful.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#31525] [Other Feature Feedback, Questions, & Ideas]
NA

**Comment by ghost** [comment]
I apologise. 

I didnt know what this is, this seems to be a good feature. 

Can you bring in developers for https://github.com/imvetri/ui-editor and rescue it. 

I'm starting my next project called visualiser. 

Your street neighbour,
Vetrivel Shanmugam.

**Comment by OlgaGrom** [comment]
Hi
